Modules/_interpqueuesmodule.c

```diff
@@ -1589,31 +1607,25 @@ queuesmod_list_all(PyObject *self, PyObject *Py_UNUSED(ignored))
     return ids;
 }
 
-PyDoc_STRVAR(queuesmod_list_all_doc,
-"list_all() -> [(qid, unboundop, fallback)]\n\
-\n\
-Return the list of IDs for all queues.\n\
-Each corresponding default unbound op and fallback is also included.");
+/*[clinic input]
+_interpqueues.put
+    qid: qidarg
+    obj: object
+    unboundop as unboundarg: int = -1
+    fallback as fallbackarg: int = -1
+
+Add the object's data to the queue.
+[clinic start generated code]*/
 
 static PyObject *
-queuesmod_put(PyObject *self, PyObject *args, PyObject *kwds)
-{
-    static char *kwlist[] = {"qid", "obj", "unboundop", "fallback", NULL};
-    qidarg_converter_data qidarg = {0};
-    PyObject *obj;
-    int unboundarg = -1;
-    int fallbackarg = -1;
-    if (!PyArg_ParseTupleAndKeywords(args, kwds, "O&O|ii$p:put", kwlist,
```

Comment:
There is a bug here. The format string is for 5 parameters, but only 4 variables are provided. Please fix it before conversion to Argument Clinic, it should be backported.